backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/model/interceptor/PP_OrderCandidate_PP_Order.java

```diff
@@ -43,7 +43,7 @@ public class PP_OrderCandidate_PP_Order
 	private void propagateQtyProcessed(@NonNull final I_PP_OrderCandidate_PP_Order ppOrderAllocation)
 	{
 		final PPOrderCandidateId ppOrderCandidateId = PPOrderCandidateId.ofRepoId(ppOrderAllocation.getPP_Order_Candidate_ID());
-		ppOrderCandidateService.updateOrderCandidateAfterCommit(ppOrderCandidateId);
+		ppOrderCandidateService.updateOrderCandidateById(ppOrderCandidateId);
```

Comment:
what if there are multiple I_PP_OrderCandidate_PP_Order updated in this trx?
now it will behave?

Reply:
agreed to move this to a follow-up.